docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1220,9 +1220,81 @@ The JavaBatch user is configured in server.xml and the bulkdata.json:
 	</security-role>
 </authorization-roles>
 ```
-Note: The user referenced in the bulkdata.json must have a role of at least batchSubmitter.
-
-By default, in-memory Derby database is used for persistence of the JavaBatch Jobs. Instruction is also provided in "Configuring a Liberty Datasource with API Key" section of the DB2OnCloudSetup guide to configure DB2 service in IBM Clouds as JavaBatch persistence store. Liberty JavaBatch framework creates DB schema and tables automatically by default for both approaches, and the configured database is created automatically, if the in-memory Apache Derby approach is used.   
+Note: The user referenced in the fhir-server-config.json must have a role of at least batchSubmitter.
+
+By default, in-memory Derby database is used for persistence of the JavaBatch Jobs as configured in batchDs.xml. Instruction is also provided in "Configuring a Liberty Datasource with API Key" section of the DB2OnCloudSetup guide to configure DB2 service in IBM Clouds as JavaBatch persistence store. Liberty JavaBatch framework creates DB schema and tables automatically by default for both approaches, and the configured database is created automatically, if the in-memory Apache Derby approach is used.
+
+You can also choose to use postgresql or other RDBMS as your Job repository, if taking this approach, you will need to generate the DDL for the job tables first following [IBM Websphere Liberty Batch - Job Repository Configuration](https://www-03.ibm.com/support/techdocs/atsmastr.nsf/WebIndex/WP102716) , then make necessary changes to the generated DDL to make it valid for your RDBMS, and then create the job database, create javabatch user, create jbatch schema and then run the DDL to generate the job tables and enable it in batchDs.xml let's use postgresql as an example:
+
+(1) Generate DDL and modify it according to your RDBMS.
+
+``` shell
+ ./ddlGen generate fhir-server
+```
+Following is a modified version for postgresql:
+
+``` shell
```

Comment:
I think this should be a file in fhir-persistence-schema

Reply:
Great point.  The webapp makes sense to me.

Reply:
OK, will add this DDL to the webapp project.

Reply:
done